src/Common/tests/System/PlatformDetection.cs

```diff
@@ -14,6 +14,7 @@ public static class PlatformDetection
     {
         public static bool IsWindows { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         public static bool IsWindows7 { get; } = IsWindows && GetWindowsVersion() == 6 && GetWindowsMinorVersion() == 1;
+        public static bool IsWindows8 { get; } = IsWindows && GetWindowsVersion() == 6 && (GetWindowsMinorVersion() == 1 || GetWindowsMinorVersion() == 2);
```

Comment:
Now on Win8 IsWindows7 will also be true - this should be fixed or at least variable name changed to IsAtLeastWindows7